src/app/components/course-card/CourseCardComponent.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import { Colors } from "../../common/assets";
+import { ICourseCardProps } from "./models";
+import {
+  Description,
+  Title,
+  Image,
+  Wrapper,
+  Content,
+  ParamsComponent,
+  Button,
+  DescriptionText,
+} from "./styles";
+
+export const CourseCardComponent: React.FC<ICourseCardProps> = ({
+  id,
+  title,
+  imageLink,
+  description,
+  lessonsCount,
+  skills,
+  rating,
+  onClickFunction,
+}) => {
+  return (
+    <Wrapper>
+      <Title>
+        <div style={{ margin: "5px 0px 0px 20px" }}>{title}</div>
```

Comment:
In my opinion, there is no need to use embedded styles. Better to move it to all styles block.

Reply:
As I understood you suggest to change div block with customized with styled components one and move styles there, am I right? It is not bad idea, but I prefer to implement such simple styling as inline one because, to my mind, there is just complication with separate component with styling, where it can be inlined.

Reply:
Я розумію, чому при використанні styled components інколи хочеться уникнути створення окремого компонента для блоку, коли стилі доволі прості. Небезпека від використання вбудованих стилів здається менш очевидною зі styled components.
Але я згоден з коментарем вище, і рекомендую все ж уникати їх використання.